packages/blockchain-wallet-v4-frontend/src/modals/Transactions/DownloadTransactions/index.tsx

```diff
@@ -32,32 +33,27 @@ type LinkStatePropsType = {
 }
 type LinkDispatchPropsType = {
   clearTransactions: () => void
-  fetchTransactions: (
-    address: string,
-    startDate: string,
-    endDate: string
-  ) => void
-  initForm: (formDefaults: {
-    end: moment.Moment
-    from: 'all'
-    start: moment.Moment
-  }) => void
+  fetchTransactions: (address: string, startDate: string, endDate: string) => void
+  initForm: (formDefaults: { end: moment.Moment; from: 'all'; start: moment.Moment }) => void
 }
 type Props = OwnProps & LinkDispatchPropsType & LinkStatePropsType
 
 class DownloadTransactionsModal extends Component<Props, StateProps> {
-  state: StateProps = { filename: '', generating: false }
+  constructor(props: Props) {
+    super(props)
+    this.state = { filename: '', generating: false }
+  }
 
   componentDidMount() {
     const { initForm } = this.props
     initForm({
-      from: 'all',
       // @ts-ignore
-      start: moment()
-        .startOf('day')
-        .subtract(7, 'day'),
+      end: moment().endOf('day'),
+
+      from: 'all',
+
       // @ts-ignore
```

Comment:
Why is this being ignored?

Reply:
it's some old code, will check can I remove it